src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,286 @@
+package duke;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * The Parser class stores all responses that Duke will display on the
+ * user's machine. The Parser class also processes user commands such as
+ * mark done, mark undone, add and delete tasks from the list of tasks.
+ *
+ * @author Muhammad Reyaaz
+ * @version %I% %G%
+ * @since 11
+ */
+class Parser {
+    //Greetings
+    static String INITIAL_GREETING = "Welcome to Manchester United.";
+    static String FINAL_GREETING = "SUIII, Bye";
+    //Command to display for various user input
+    static String MARK_COMMAND = "SUI, I have marked this task from the training room: ";
+    static String UNMARK_COMMAND = "SUI, I have unmarked this task from the training room: ";
+    static String DELETE_COMMAND = "Tasks successfully deleted. SUI.";
+    static String FIND_COMMAND = "Here are matching tasks in your list";
+    //Whitelist commands
+    static String SHOW_TASKS = "list";
+    static String TERMINATE = "exit";
+    static String MARK = "mark";
+    static String UNMARK = "unmark";
+    static String TODO = "todo";
+    static String DEADLINE = "deadline";
+    static String EVENT = "event";
+    static String DELETE = "delete";
+    static String FIND = "find";
+    //List starts numbering from 1 not from 0
+    static int DECREMENT = 1;
+    //Symbols to display the state of the task to the user beside the type of Task
+    static String MARK_SYMBOL = "X";
+    static String TODO_SYMBOL = "T";
+    static String DEADLINE_SYMBOL = "D";
+    static String EVENT_SYMBOL = "E";
+    //Blacklist
+    static List<String> INVALID_COMMANDS = Arrays.asList("blah", "todo", "deadline", "event");
```

Comment:
Would be good to label these strings as final since they follow the variable naming conventions for constants.